docs/design/coreclr/botr/clr-abi.md

```diff
@@ -705,3 +705,7 @@ The general rules outlined in the [System V x86_64 ABI](https://software.intel.c
 | %xmm2-%xmm7  | used to pass floating point arguments   | No                |
 | %xmm8-%xmm15 | temporary registers                     | No                |
 ```
+
+# x86 Calling convention specifics
+
+The managed x86 calling convention uses a variant of the Windows x86 "fastcall" calling convention. The first two register-eligible arguments are passed in ecx and edx respectively. 1, 2, or 4 byte integer primitives and structs that trivially wrap a 4-byte primitive integer are eligible for registers. Structs that do not trivially wrap a 4-byte primitive integer or a register-eligible struct are passed on the stack. The this argument and the hidden arguments mentioned earlier in this document are also eligible for registers. All non-register eligible arguments are pushed on the stack in argument order. This is in contrast to the unmanaged calling conventions, cdecl, stdcall, thiscall, and fastcall, which push the arguments in reverse order onto the stack.
```

Comment:
Could you please start with information that x86 windows managed and native calling conventions are different in contrast with other platforms (including x86 Unix)?

Reply:
The x86 managed calling convention for Unix is not the same as the native Unix x86 calling convention. They also have the differences I've listed. I'll add a blurb calling out the difference for x86 vs other architectures.